subgraphs/harvest-finance/tests/utils/prices/uniswap.test.ts

```diff
@@ -0,0 +1,133 @@
+import { Address, BigInt } from '@graphprotocol/graph-ts'
+import { assert, describe, test } from 'matchstick-as'
+import { constants } from '../../../src/utils/constants'
+import { uniswap } from '../../../src/utils/pricies/uniswap'
+import {
+  mockERC20,
+  mockUniswapRouter,
+  mockUniswapRouter_factory,
+} from '../../controller-utils'
+import { helpers } from '../helpers'
+
+const tokenAddress = Address.fromString(
+  '0x6b175474e89094c44da98b954eedeac495271d0f'
+)
+
+const wethAddress = constants.WETH_ADDRESS
+
+describe('prices', () => {
+  describe('uniswap', () => {
+    describe('isLpToken', () => {
+      describe('when tokenAddress is lpToken', () => {
+        test('returns true', () => {
+          mockUniswapRouter_factory(tokenAddress)
+
+          const result = !!uniswap.isLpToken(tokenAddress)
```

Comment:
The function isLpToken always returns a boolean so there is no need for the !! operator.

Reply:
the problem is that it return another type: boolean / bool